core/chains/evm/chain_set.go

```diff
@@ -91,14 +103,61 @@ func (cll *chainSet) Default() (Chain, error) {
 	return cll.Get(cll.defaultID)
 }
 
+func (cll *chainSet) Configure(id *big.Int, enabled bool, config types.ChainCfg) (types.Chain, error) {
+	// Update configuration stored in the database
+	bid := utils.NewBig(id)
+	dbchain, err := cll.orm.UpdateChain(*bid, enabled, config)
+	if err != nil {
+		return types.Chain{}, err
+	}
+	// TODO: replace with math.MaxInt once we make go 1.17 mandatory
```

Comment:
Is it out yet? Perhaps it can be done in this PR?

Reply:
It's out but it came out just weeks ago. I know you needed some of the changes for tests to work? Wasn't sure what our policy was regarding golang version bumps

Reply:
dont see any reason against bumping asap. All you need to do is update the builder image